go.mod

```diff
@@ -4,8 +4,8 @@ go 1.16
 
 require (
 	github.com/hashicorp/go-memdb v1.3.2
-	github.com/hashicorp/terraform-plugin-framework v0.5.0
+	github.com/hashicorp/terraform-plugin-framework v0.5.1-0.20220208154921-bf0240fa43e1
 	github.com/hashicorp/terraform-plugin-go v0.7.1
-	github.com/hashicorp/terraform-plugin-mux v0.4.0
+	github.com/hashicorp/terraform-plugin-mux v0.5.1-0.20220215203326-d8fdec82712e
```

Comment:
Temporarily pointing at github.com/hashicorp/terraform-plugin-mux@bflad-PreparedConfig (https://github.com/hashicorp/terraform-plugin-mux/pull/54) to fix `tf5muxprovider` and `tf6muxprovider` tests. We can wait to merge this until terraform-plugin-mux@v0.5.1 release has been published.